### PR TITLE
Add grammari-x extension

### PR DESCRIPTION
**Pull request by hakob8956**
## Description
In this extension, you can fix your grammar and rewrite your sentences more effectively.
Before using this extension, you need to pass the OpenAI API key to make the extension work.
## Screencast
improve grammer:

https://github.com/raycast/extensions/assets/25252241/68e71d1b-6ecf-4174-8ad0-1a5b967624bb

History and saved functions.

https://github.com/raycast/extensions/assets/25252241/fdf6c285-435e-4a6d-ad37-01eee94d977e

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/grammari-x/package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "grammari-x",
+  "title": "Your Writing's Best Friend",
```
Comment (pernielsentikaer) [inline extensions/grammari-x/package.json:4]:
```suggestion
  "title": "GrammariX",
```

File: extensions/grammari-x/package.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  "icon": "icon.png",
+  "author": "hakob_nersesian",
+  "categories": [
+    "Communication"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "grammariX",
+      "title": "GrammariX",
```
Comment (pernielsentikaer) [inline extensions/grammari-x/package.json:15]:
```suggestion
      "title": "Improve Writing",
```

File: extensions/grammari-x/package.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "grammari-x",
+  "title": "Your Writing's Best Friend",
+  "description": "In this extension, you can fix your grammar and paraphrase your sentences more effectively.",
```
Comment (pernielsentikaer) [inline extensions/grammari-x/package.json:5]:
```suggestion
  "description": "Your Writing's Best Friend. In this extension, you can fix your grammar and paraphrase your sentences more effectively.",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
What do you say about the following corrections?

**Comment by hakob8956** [comment]
> What do you say about the following corrections?

It seems reasonable. I'll commit to it ;)

**Comment by hakob8956** [comment]
@pernielsentikaer 
For future implementation, as you can see in the upper right-hand corner, there is a dropdown that is specifically related to the 'tone changer.' However, it appears that there is no feature available to create a dropdown-like list item. Due to this limitation, I have positioned it globally after the search bar. It would be preferable if I could trigger this dropdown when the 'tone changer' list item is selected. Unfortunately, there is no implemented event for such an action. I am aware that I could create separate commands for each list item, but this doesn't seem like an optimal solution. I would prefer to have a single command for all list items due to certain reasons. If you have any valuable suggestions in this context, I am open to listening.

**Comment by hakob8956** [comment]
@pernielsentikaer 
Added history and saved answer functionality

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**